pkg/scheduler/framework/runtime/framework.go

```diff
@@ -313,6 +313,13 @@ func NewFramework(r Registry, profile *config.KubeSchedulerProfile, stopCh <-cha
 		// Update ClusterEventMap in place.
 		fillEventToPluginMap(p, options.clusterEventMap)
 	}
+	if len(f.extenders) > 0 {
```

Comment:
Same as v1.26, it's something different from the original PR.

https://github.com/kubernetes/kubernetes/pull/122043#discussion_r1405340648